source/install/desktop.rst

```diff
@@ -0,0 +1,56 @@
+===================================
+Desktop Applications 
+===================================
+
+Mattermost desktop applications are available for Windows, Mac and Linux operation systems. 
+
+They support all the features of the web experience, plus: 
+
+- Connect to multiple Mattermost servers from a single interface, and switch with shortcut keys.
+- Auto-start Mattermost when a user logs into their machine
+- (Windows) Add Mattermost to Start menu, taskbar and system tray
+- (Mac) Add Mattermost to the applications Dock
+- (Linux) ``Desktop Entry`` for the application to more easily `integrate into a desktop environment <https://wiki.archlinux.org/index.php/Desktop_entries>`_
+
+Below is a list of additional resources:
+
+ - `Guide for configuring your desktop app experience <https://docs.mattermost.com/help/apps/desktop-guide.html>`_
+ - `Changelog <https://docs.mattermost.com/help/apps/desktop-changelog.html>`_
+ - Contributor’s guide (coming soon)
+ - `Source code <https://github.com/mattermost/desktop>`_
+
+You can `download the apps directly from our downloads page <https://about.mattermost.com/downloads/>`_. You may also use the following installation guides for Windows, Mac and Linux:
+
+**Windows 7+:**
```

Comment:
Propose title **Windows 10+, Windows 8.1+, Windows 7+**  to make it clear we're supporting the different OS releases, not just the versions of Windows 7